py/picca/converters.py

```diff
@@ -320,6 +324,13 @@ def desi_from_ztarget_to_drq(in_path,
     for key in ['RA', 'DEC']:
         cat[key] = cat[key].astype('float64')
 
+    # apply error to z
+    if gauss_sigma_v is not None:
+        c = 299792
```

Comment:
@SamYoules - we definitely have the speed of light defined somewhere in Picca, it would be better to not hardcode values like this in the code.

Reply:
Thanks Andreu. This is done now.